proxy/v1/session/update-default/update-default.5.x.php

```diff
@@ -13,9 +13,6 @@
 
 $session = $twilio->proxy->v1->services("KSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                              ->sessions("KCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
-                             ->update(array(
-                                          'dateExpiry' => new \DateTime('2008-1-2')
-                                      )
-                             );
+                             ->update(array("uniqueName" => "uniqueName"));
 
```

Comment:
@po5i I have a pending PR to fix this code sample definition... please manually remove this file for now.